mrp_food/models/product_product.py

```diff
@@ -62,6 +62,7 @@ def _onchange_date_last_statement_price(self):
         for product in self:
             product.date_last_statement_price = fields.Date.today()
 
+    @api.depends("product_seasonality_ids")
```

Comment:
```suggestion
    @api.depends("product_seasonality_ids.date_start", "product_seasonality_ids.date_end")
```
je crois que c'est plus précis car ça dépend concrètement de ces deux champs. (après, comme c'est un champ non storé, peut-etre que cette modification que je propose ne sert à rien)...

Reply:
hmm sur la vue article où l'on veut que ça se déclenche, ça dépend du choix de la saisonnalité. Si on change la date de début ou fin, ça changera bien la saison du produit grâce au compute, yes vu que c'est pas storé :)